### ISSUE TITLE
Color Contrast: Many Pages: The white text on red fails contrast ratio

### ISSUE THREAD

**Issue by nbhardwaj-del** [Expensify/App#76916]
## Action Performed:
Prerequisites:
- the user is signed in

1. Using Chrome, open the URL - https://new.expensify.com/
2. Tab to focus on the Account>Security button and activate it.
3. Tab to focus on the Close account under Security options and activate it.
4. Measure the color contrast for the white text 'Close' on the red button.

Other Occurrences:
1. On Workspaces - Delete Workspace: the button 'Delete'
2. On Create Expense flow - Delete Expense dialog

## Expected Result:
Text and background colors should have a contrast ratio of at least 4.5:1 for normal text. The white text on the red background should appear clearly distinguishable to all users, including those with low vision or color vision deficiencies.

## Actual Result:
The “Close account” button uses text color #E7ECE9 (rgb(231, 236, 233)) on a background color #F25730 (rgb(242, 87, 48)), producing a contrast ratio of approximately 3.4:1. This contrast is below the required 4.5:1 ratio, making the text difficult to read for users with low vision or color blindness.

HTML:
<button aria-label="" role="button" data-selection-scrapper-hidden-element="true" data-tag="pressable" data-listener="Enter" tabindex="0" class="css-g5y9jx r-1otgn73 r-1loqt21 r-1awozwy r-1777fci r-1mdbw0j r-wk8lta r-18c69zk r-19yznuf r-64el8z r-u9wvl5 r-1qjhsva r-1yadl64 r-9ffhgg r-lrvibr" id="" type="button" data-component="GenericPressable" data-element="Pressable" data-source-file="BaseGenericPressable.tsx"><div dir="auto" data-selection-scrapper-hidden-element="true" class="css-146c3p1 r-dnmrzs r-1udh08x r-1udbk01 r-3s2u2q r-1iln25a" style="color: rgb(255, 255, 255); font-size: 15px; text-align: center; font-family: &quot;Expensify Neue&quot;, &quot;Windows Segoe UI Emoji&quot;, &quot;Noto Color Emoji&quot;, &quot;Custom Emoji Font&quot;; font-style: normal; font-weight: 700; pointer-events: none; flex-shrink: 1; line-height: 20px; padding-bottom: 1px;">Close account</div></button>

CSS:
.r-1qjhsva {
background-color: rgb(242, 87, 48);

## Area issue was found in:
Settings-Security

## Failed WCAG checkpoints
WCAG 1.4.3 Contrast (Minimum)
https://www.w3.org/WAI/WCAG22/Understanding/contrast-minimum

## User impact:
People with low vision often have difficulty reading text that does not contrast with its background. This can be exacerbated if the person has a color vision deficiency that lowers the contrast even further. Providing a minimum luminance contrast ratio between the text and its background can make the text more readable even if the person does not see the full range of colors. It also works for the rare individuals who see no color.

## Suggested resolution:
Use a darker red color for the background to meet a contrast ratio of at least 4.50:1.
https://webaim.org/articles/contrast/#sc143

## Environment(s)
Windows Windows 11 - Chrome

## Screenshots/Videos
<img width="1916" height="970" alt="Image" src="https://github.com/user-attachments/assets/01b7128a-5057-4cee-9769-1bb721ba877f" />

**Version Number:** NA
**Reproducible in staging?:** NA
**Reproducible in production?:** Yes
**Email or phone of affected tester (no customers):** NA
**Logs:** https://stackoverflow.com/c/expensify/questions/4856 NA
**Notes/Photos/Videos:** Any additional supporting documentation Yes
**Expensify/Expensify Issue URL:** https://platform.applause.com/company/12102/products/36605/community-issues/6994572
**Issue reported by:** NA
**Slack conversation:** NA

**Comment by rushatgabhane** [comment]
This issue was retested in **BFV cycle 544083 (Web)** on 2026-04-28 and is **Not Fixed**.

### Screenshots / Videos
- https://platform.applause.com/services/links/v1/external/7c70184ba453f72f7d5ce1d70b1ebcbe9607135f02cadf895f6d6684fec57ea1
- https://platform.applause.com/services/links/v1/external/9f0feb17ddfc092b5ae73a5fefdc674597a45552ba61628d8945f17a79ca8930
- https://platform.applause.com/services/links/v1/external/2625c8f9b432bf5e05e5f62b1329ff0f8d7b8e90468c807b795e99bea8592968

### Retest notes
Not fixed: The issue remains. The “Close account” button  text color #E7ECE9 (rgb(231, 236, 233)) on a background color #F25730 (rgb(242, 87, 48)), producing a contrast ratio of approximately 3.4:1.

Not fixed:
Other Occurrences: The issue is not fixed for the following occurrences as well:
1. On Workspaces - Delete Workspace: the button 'Delete'
2. On Create Expense flow - Delete Expense dialog

**Comment by dubielzyk-expensify** [comment]
This doesn't seem to be using the high contrast mode?
